week-03/main.js

```diff
@@ -43,10 +61,38 @@ var list = [1, 2, 3, 4, 5];
 
 var list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+var answer = document.querySelector('#answer');
+
+var filteredOdd = list.filter(function(a) {
+  if (a % 2 !== 0) {
```

Comment:
This just needs to return a true or false statement, so you could simply say `return a % 2 !== 0`